evals/index.eval.ts

```diff
@@ -2,30 +2,47 @@ import { Eval } from "braintrust";
 import fs from "fs";
 import path from "path";
 import process from "process";
-import { EvalFunction } from "../types/evals";
+import { EvalCategory, EvalFunction } from "../types/evals";
 import { AvailableModel } from "../types/model";
 import { EvalLogger, env } from "./utils";
 
-const models: AvailableModel[] = ["gpt-4o", "claude-3-5-sonnet-latest"];
+const MODELS: AvailableModel[] = process.env.EVAL_MODELS.split(",").map(
```

Comment:
let's do `const DEFAULT_EVAL_MODELS = process.env.EVAL_MODELS ? 
process.env.EVAL_MODELS.split(",") : ["gpt-4o", "claude-3-5-sonnet-latest"]`

and then reference `DEFAULT_EVAL_MODELS` directly. SAme with below. Good practice to not directly reference env vars